components/ProjectsView.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div class="px-4">
+    <ul class="project-list" role="list" >
+      <li v-for="project in projects" :key="project.slug">
+        <div class="flex-none w-32">
+          <p class="project-name">{{ project.name }}</p>
+          <p class="project-type">Type: {{ project.type }}</p>
+        </div>
+        <ProjectButtons :project="project" />
+        <div class="w-64">
+          <span class="text-sm font-medium">Project usage: <b>{{ project.usage }} </b></span>
+        </div>
+        <div class="hidden sm:flex sm:flex-col sm:items-end">
+          <button type="button" class="ml-auto flex-shrink-0 rounded-full bg-gray-800 p-1 text-gray-400 hover:text-white" @click="deleteProject(project.slug)">
+            <span class="sr-only">Delete</span>
+            <TrashIcon class="h-6 w-6" aria-hidden="true" />
+          </button>
+        </div>
+      </li>
+    </ul>
+    <div class="flex py-2 justify-stretch">
+      <button
+        class="w-full rounded-md px-3 py-3 text-med text-white font-medium "
```

Comment:
There isn't a `text-med` I don't believe, at least in tailwind. `text-base` is regular or "medium."